app/views/users/list.html.erb

```diff
@@ -56,8 +56,8 @@
                       <li><a style="color: #808080; cursor: no-drop;"><%= user.nodes.count %> Wiki Edits</a></li>
                       <% if current_user && current_user.can_moderate? %>
                          <% if user.status == 0 %>
-                           <li class="btn-success"><a  style="color: white;" href="/profile/<%= user.username %>"><i class="fa fa-check-circle fa-white"></i> Unban on profile</a></li>
-                           <i class='fa fa-ban' style="color:#a00;"></i> <%= translation('users.list.banned') %>
+                           <li class="btn-success m-2" style="border-radius: 2px;"><a class="p-1 text-center w-100" style="color: white;" href="/profile/<%= user.username %>"><i class="fa fa-check-circle fa-white"></i> Unban on profile</a></li>
++                           <p class="text-center m-0"><i class='fa fa-ban' style="color:#a00;"></i> <%= translation('users.list.banned') %></p>
```

Comment:
Hi @Beckeruni, seems you added unwanted `+` at the beginning of this line. Kindly check. Everything else looks good.
Thanks for working on this